From e9c7fe8509282d39f8991b62257c8103c9bfe4fc Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 7 Dec 2012 16:48:09 +0800
Subject: [PATCH 373/505] ENGR00236189: mxc_v4l2_test: Remove IMX6Q out of
 mxc_v4l2_test Makefile CSI list

- csi_v4l2_capture/overlay test should not be compiled for IMX6Q
So remove IMX6Q out of CSI_LIST
- NEED_PXP_LIB_LIST will add -lpxp into the compile option.
So remove IMX6Q out of NEED_PXP_LIB_LIST

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 test/mxc_v4l2_test/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_v4l2_test/Makefile b/test/mxc_v4l2_test/Makefile
index baa4487..ff3eca5 100644
--- a/test/mxc_v4l2_test/Makefile
+++ b/test/mxc_v4l2_test/Makefile
@@ -1,7 +1,7 @@
 # list of platforms which did not want this test case
 EXCLUDE_LIST:=
-CSI_LIST:=IMX25_3STACK IMX6Q IMX6S
-NEED_PXP_LIB_LIST:=IMX6Q IMX6S
+CSI_LIST:=IMX25_3STACK IMX6S
+NEED_PXP_LIB_LIST:=IMX6S
 
 # Just use pxp library to get contiguous physical memory for USERPTR testing
 # However, be noticed that this is not the only way.
-- 
1.8.0

